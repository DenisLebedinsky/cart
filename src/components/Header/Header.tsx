import * as React from "react";

import css from "./_Header.module.scss";

import { HeaderProps } from "./Header.type";

const Header: React.FC<HeaderProps> = ({ title }) => (
  <h1 className={css.header}>{title}</h1>
);

export default Header;
