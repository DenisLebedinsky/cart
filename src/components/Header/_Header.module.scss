.header {
  font-size: 48px;
  font-family: Roboto, sans-serif;
  text-transform: uppercase;
  text-align: center;
}
