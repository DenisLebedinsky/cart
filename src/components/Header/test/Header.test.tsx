import * as React from "react";
import { shallow, ShallowWrapper } from "enzyme";

import Header from "../Header";

describe("Header", () => {
  let wrapper: ShallowWrapper<typeof Header>;

  const title = "marklar";

  beforeEach(() => {
    wrapper = shallow(<Header title={title} />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should have marklar as textContent", () => {
    expect(wrapper.contains(title)).toBe(true);
  });
});
