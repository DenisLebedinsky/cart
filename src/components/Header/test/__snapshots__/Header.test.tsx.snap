// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<h1
  className="header"
>
  marklar
</h1>
`;
